changelog/changes/crash-prime-aboard.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "New string padding functions"
```

Comment:
We used `trim_start` and `trim_end`, as it is agnostic to the direction of writing. Should these be called `pad_start` and `pad_end` instead?

Reply:
Yes, good point. That said, I find "left" and "right" a lot more intuitive. Any objections to renaming the trim functions to `trim_left` and `trim_right`?

Reply:
Okay, given that other languages do this as well, I'll go with `pad_start` and `pad_end`.